src/ixdat/measurements.py

```diff
@@ -130,11 +181,8 @@ def from_dict(cls, obj_as_dict):
             technique_class = cls
         try:
             measurement = technique_class(**obj_as_dict)
-        except TypeError:
-            raise TechniqueError(
-                f"Can't initiate a {technique_class} with the"
-                f"kwargs {list(obj_as_dict.keys())}"
-            )
+        except Exception:
```

Comment:
Here I think it was because I thought I should handle it but didn't know what the error would be so wanted to figure that out first and then add code. I do that sometimes. But seeing as I never added handling it probably wasn't necessary to handle. Will delete.

The rest of this comment: Love it! Thanks for including this kind of python research here, much appreciated :D 

I had always wondered what the "During handling of the above exception, another exception occurred" message was about - I saw it a few too many times from spitze ;)

Reply:
deleted.

Reply:
Uuuuh, that hurts a little. We have fewer of these nowadays ;)